requirements.txt

```diff
@@ -0,0 +1,4 @@
+chardet==3.0.4
```

Comment:
Could this `requirements.txt` be re-used by `setup.py`? Maybe using `pbr`.

Also, this is a library and will be included in other projects. It would be better to pin to a range of versions.
I think `chardet<4` would be good enough.